WordPress/Classes/ViewRelated/Jetpack/Branding/Menu Card/JetpackBrandingMenuCardPresenter.swift

```diff
@@ -11,8 +11,8 @@ class JetpackBrandingMenuCardPresenter {
 
     private let remoteConfigStore: RemoteConfigStore
     private let persistenceStore: UserPersistentRepository
-    private let featureFlagStore: RemoteFeatureFlagStore
     private let currentDateProvider: CurrentDateProvider
+    private let phase: JetpackFeaturesRemovalCoordinator.GeneralPhase
```

Comment:
The unit tests for this class are failing because the phase doesn't get updated when you do `remoteFeatureFlagsStore.removalPhaseX = true`

Reply:
Now that I think about this, other than the tests, if the phase changes, we'll have to refresh the presenter to carry the correct value. So given that, I think it's best to revert this change altogether.

Reply:
Updated in 9dc4c28fb15aad99889aa317690610fc772dd971
Let me know what you think!